### PR TITLE
Update: `WikidPad.Wikidpad` version 2.4alpha01

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21195)

### CODE REVIEW

File: manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WikidPad.Wikidpad
```
Comment (ItzLevvie) [inline manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.installer.yaml:4]:
```suggestion
PackageIdentifier: WikidPad.Wikidpad.Alpha
```

File: manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: WikidPad.Wikidpad
```
Comment (ItzLevvie) [inline manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.yaml:4]:
```suggestion
PackageIdentifier: WikidPad.Wikidpad.Alpha
```

File: manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WikidPad.Wikidpad
```
Comment (ItzLevvie) [inline manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: WikidPad.Wikidpad.Alpha
```

File: manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.installer.yaml:4]: You'll have to update the folder names and file names.

File: manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+PackageVersion: 2.4alpha01
+PackageLocale: en-US
+Publisher: WikidPad
+PublisherUrl: http://wikidpad.sourceforge.net
+PublisherSupportUrl: https://github.com/WikidPad/WikidPad/issues
+#PrivacyUrl: 
+Author: Michael Butscher, Jason Horman, Gerhard Reitmayr
+PackageName: WikidPad
```
Comment (ItzLevvie) [inline manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.locale.en-US.yaml:12]:
```suggestion
PackageName: WikidPad (Alpha)
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/w/WikidPad/WikidPad/2.4alpha01/WikidPad.Wikidpad.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/21195#pullrequestreview-708434799

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
All `.yaml` files need to have their names be changed from `WikidPad.Wikidpad` to `WikidPad.Wikidpad.Alpha`

**Comment by vedantmgoyal2009** [comment]
I got confused between these two PRs. Both of them failed.

**Comment by vedantmgoyal2009** [comment]
Can anyone tell why this error is occurring?
image

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie please open this PR, as the WikidPad conflict has now been corrected, I want to merge it. Please.

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie @ItzLevvie2 Please 🙏

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie @ItzLevvie2 Why you have closed my PR explicitly, you could have commented and I would have closed myself.

**Comment by vedantmgoyal2009** [comment]
@denelon can you open this PR? i guess you will be able to do as you have admin rights...

**Comment by ItzLevvie** [comment]
Sorry, I didn't see this PR. @denelon won't be active until next week.

Just open up another PR with your changes using GitHub Desktop. My mind thought that `PullRequest-Error` was `Validation-Unattended-Failed` or `Status: no recent activity` or both which is why it was added to the pile of "PRs to be closed as they were invalid".

**Review by ghost — COMMENTED** [review]
This has been added with https://github.com/microsoft/winget-pkgs/commit/d02f98806666ab3b81f755741290a41c11a3f6e1.

**Pull request closed without merging**